### DISCUSSION TITLE
How to trigger the CI when Dependabot open a pull request?

### DISCUSSION THREAD

**Discussion by diegobernardes** [community/community#25907] [Actions]
<p>This <a href="https://github.com/Nitro/markdown-link-check/pull/54" rel="nofollow noopener">pull request</a> does not trigger the <a href="https://github.com/Nitro/markdown-link-check/blob/284b55b5cd8a6a4ed2d21449115b511260ddc2fa/.github/workflows/ci.yml" rel="nofollow noopener">CI</a>. Do you know what’s wrong?</p>
<p>I want to avoid adding the <code>pull_request</code> at the trigger to avoid the CI to run twice when pushing the code and create a pull request.</p>

**Comment by NicoNBW** [comment]
<p>Hi <a class="mention" href="/diegobernardes">@diegobernardes</a><br>
You could add  <code>pull_request</code>  event and <strong>specify it’s Activity types</strong>. In your scenario, you could remove <code>opened</code> ,  <code>synchronize</code> Activity types in order to avoid the CI to run twice when push the code and create a pull request. You could keep other Activity types listed <a href="https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request" rel="nofollow noopener">here</a> according to your needs.<br>
For example :</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - '*'
      - '!master'
  pull_request:
    types: [assigned, reopened]

jobs:
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
</code></pre>

**Comment by diegobernardes** [comment, marked as answer]
<p>Thanks! That works. But I actually found a way simpler solution, the problem was with the branch filter pattern. Instead of using <code>*</code> I had to use <code>**</code> and that solved the problem.</p>
